src/Components/Books.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import Form from './Form';
+import BookList from './BookList';
+import './Books.css';
+
+function Books() {
+  return (
+    <div className="booksMain">
+      <BookList />
```

Comment:
- [ ] It is required to pass data prop to this BookList component and use it to render title and author in its stand-alone.
e.g
```
   <BookList book={book}  />
```

- [OPTIONAL] The proper name for this component should been `BookList` component and name the component you call  inside here `Book`.